core/src/main/scala/com/lightbend/paradox/template/PageTemplateSystem.scala

```diff
@@ -23,7 +23,7 @@ import org.stringtemplate.v4.{ STErrorListener, STRawGroupDir, ST }
 import collection.concurrent.TrieMap
 
 object CachedTemplates {
-  def apply(dir: File, templateName: String): PageTemplate = {
+  def apply(dir: File, templateName: String = PageTemplate.DefaultName): PageTemplate = {
```

Comment:
Oh yes my bad, you're totally right.

What I had in my mind actually is that the directory template is always the same, it's why I've implemented like this... But sure, the directory should be taken into account in the cache.